src/samples/UWP/Uno.Themes.Samples.Shared/Content/Controls/ListViewSamplePage.xaml.cs

```diff
@@ -1,32 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Runtime.InteropServices.WindowsRuntime;
-using Uno.Themes.Samples.Entities;
-using Windows.Foundation;
-using Windows.Foundation.Collections;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Controls.Primitives;
-using Windows.UI.Xaml.Data;
-using Windows.UI.Xaml.Input;
-using Windows.UI.Xaml.Media;
-using Windows.UI.Xaml.Navigation;
-
 // The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
 
-namespace Uno.Themes.Samples.Content.Controls
-{
-	/// <summary>
-	/// An empty page that can be used on its own or navigated to within a Frame.
-	/// </summary>
-	[SamplePage(SampleCategory.Controls, "ListView", Description = "Represents a control that displays data items in a vertical stack.", DocumentationLink = "https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.ListView?view=winui-3.0")]
-	public sealed partial class ListViewSamplePage : Page
+namespace Uno.Themes.Samples.Content.Controls;
+
+/// <summary>
+/// An empty page that can be used on its own or navigated to within a Frame.
+/// </summary>
+[SamplePage(SampleCategory.Controls, "ListView", Description = "Represents a control that displays data items in a vertical stack.", DocumentationLink = "https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.ListView?view=winui-3.0")]
+public sealed partial class ListViewSamplePage : Page
     {
-        public ListViewSamplePage()
+		public string Letters => "abcdefghijklmn";
```

Comment:
This is not supposed to work in WinUI, you may need to make an actual array out of this.

Reply:
it doesnt work if you to `ItemsSource="abcdefghijklmn"` but it works if you x:Bind to a `string` :) I guess because it sees it as an `IEnumerable` of `char`